### DISCUSSION TITLE
remove thumb up reaction from discussions

### DISCUSSION THREAD

**Discussion by DetachHead** [community/community#9414] [Discussions]
![image](https://user-images.githubusercontent.com/57028336/147729536-9fd497f3-edcf-4112-aa7c-6c3246a14318.png)

there's already an upvote button so the thumb up button is confusing/duplicated functionality

**Comment by ShahriarKh** [comment]
I believe removing the upvote button and keeping the :+1: makes more sense because there's also the :-1:

**Comment by rubyFeedback** [comment]
I do not have any particularly strong opinion pro or con, though I am slightly in favour of keeping the thumbs up and other mini-reactions.

But I want to raise something else. On some github trackers, such as ladybird, a thumbs up reaction is automatically removed.
The problem I have here is that I am not notified as to why it is removed. It can be a project-wide setting or a per-person setting. I would simply like to know which way.

This could be done by some info notice that could fade in or fade out or some other way e. g.:

"This project disabled user reactions on a per-issue basis". Or something like this.

Right now we don't get ANY notification as to why this does not work. I believe the user should be entitled to understand why this works on some projects but not on others. Right now I am trying to find out where this has been changed or set and I literally can not find anything. I found this here by random google search - google search is also useless now, but we also have the problem that more and more things are no longer explained to people. How should people find out why it works for some projects but not for others? A notification in some way would be really really useful here.

**Comment by Cyvadra** [comment]
👍 is just an emoji icon expressing emotions, while "Up vote" means you think it's helpful. And icons don't always appear.
